services/movie_session.py

```diff
@@ -0,0 +1,43 @@
+from django.utils.dateparse import parse_date
+from db.models import MovieSession
+
+
+def create_movie_session(movie_show_time: int |float,
+                         movie_id: id,
+                         cinema_hall_id: int) -> MovieSession:
+    movie=Movie.objects.get(id=movie_id)
+    cinema_hall=CinemaHall.objects.get(id=cinema_hall_id)
+
+    return MovieSession.objects.create(
+        movie_show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id)
+
+def get_movies_sessions(session_date: str) -> MovieSession:
+    if session_date:
+        date=parse_date(session_date)
+        return MovieSession.objects.filter(show__time__date=date)
```

Comment:
There is a logical error in the filter condition. It should be `show_time__date=date` instead of `show__time__date`.